packages/api/src/external/commonwell/patient.ts

```diff
@@ -113,17 +107,17 @@ export async function create({
 }): Promise<{ commonwellPatientId: string; personId: string } | void> {
   const { log, debug } = out(`CW create - M patientId ${patient.id}`);
 
-  const cwCreateEnabled = await validateCWEnabled({
+  const isCwEnabled = await validateCWEnabled({
     patient,
     facilityId,
-    forceCW: forceCWCreate,
+    forceCW: forceCWCreate || Config.isSandbox(),
```

Comment:
Why would we check for sandbox, if we don't have CW enabled on that env?

Reply:
This is carry-over from the current definition of validateCWEnabled:

```
  const isSandbox = Config.isSandbox();

  if (!isCommonwellEnabledForPatient(patient)) {
    log(`CW disabled for patient, skipping...`);
    return false;
  }

  if (forceCW || isSandbox) {
    log(`CW forced, proceeding...`);
    return true;
  }
```

I don't know why we're returning true when sandbox if it's disabled there...

Reply:
Conclusion from slack: leave CW enabled via force for sandbox because we mock CW for sandbox.

Reply:
I'd avoid having `forceCWCreate || Config.isSandbox()` in multiple places. Maybe we leave the `|| Config.isSandbox()` that we had on `validateCWEnabled`?

And then add a TSDoc on `validateCWEnabled` explaining why were checking for sandbox.

Reply:
Done.  I don't really agree with keeping it in the validation function, it's a shared function now and shouldn't have behavior like this IMO, but having to remember to make it true for Sandbox also isn't ideal.

Reply:
Yeah, could prob split the function at some point, but I think it's better on the validation than spread around. 👍